client/src/components/Modal.js

```diff
@@ -2,28 +2,32 @@ import Icon from 'components/Icon'
 import { Box, Button, Layer } from 'grommet'
 import * as React from 'react'
 
-export const Modal = ({ showing, setShowing, children }) => {
+export const Modal = ({ showing, setShowing, nondismissable, children }) => {
+  const dismissModal = () => {
+    if (!nondismissable) {
+      setShowing(false)
+    }
+  }
   return (
     <Box>
       {showing && (
-        <Layer
-          onEsc={() => setShowing(false)}
-          onClickOutside={() => setShowing(false)}
-        >
+        <Layer onEsc={dismissModal} onClickOutside={dismissModal}>
```

Comment:
should these be dependent on `nondismissable`

Reply:
nevermind that looks good

Reply:
They are, as part of the dismissModal function.